04/app.js

```diff
@@ -5,4 +5,73 @@ const menuItems = [
     {text: 'start', url: '/'},
     {text: 'galeria', url: '/gallery'},
     {text: 'kontakt', url: '/contact'},
-];
\ No newline at end of file
+];
+
+/* First try
+const navLocation = document.querySelector('nav');
+const menu = document.createElement('ul');
+
+const menuContainer1 = document.createElement('li');
+const menuContainer2 = document.createElement('li');
+const menuContainer3 = document.createElement('li');
+const anchor1 = document.createElement('a');
+const anchor2 = document.createElement('a');
+const anchor3 = document.createElement('a');
+
+console.log(anchor1);
+
+anchor1.setAttribute('href', '/');
+anchor2.setAttribute('href', '/gallery');
+anchor3.setAttribute('href', '/contact');
+
+anchor1.innerText = 'start';
+anchor2.innerText = 'galeria';
+anchor3.innerText = 'kontakt';
+
+menu.appendChild(menuContainer1);
+menu.appendChild(menuContainer2);
+menu.appendChild(menuContainer3);
+menuContainer1.appendChild(anchor1);
+menuContainer2.appendChild(anchor2);
+menuContainer3.appendChild(anchor3);
+
+navLocation.appendChild(menu);
+*/
+
+/* Second try
+const navLocation = document.querySelector("nav");
+
+const menuStructure = document.createRange();
+const menu = menuStructure.createContextualFragment(`
+    <ul>
+      <li><a href="/">start</a></li>
+      <li><a href="/gallery">galeria</a></li>
+      <li><a href="/contact">kontakt</a></li>
+    </ul>
+`);
+
+navLocation.appendChild(menu);
+*/
+
+const navLocation = document.querySelector("nav");
+const menu = document.createElement("ul");
+
+const menuContainer1 = document.createElement("li");
+const menuContainer2 = document.createElement("li");
+const menuContainer3 = document.createElement("li");
+const anchor1 = document.createElement("a");
+const anchor2 = document.createElement("a");
+const anchor3 = document.createElement("a");
```

Comment:
Zwróć uwagę, że `menuItems` i to ten element możemy wykorzystać do tego, aby mieć "pętle". Nie musimy tworzyć nowej tablicy. Możemy w tej pętli tworzyć kolejne `li` oraz `a`